### ISSUE TITLE
scanbox version

### ISSUE THREAD

**Issue by ytsimon2004** [jcouto/sbxreader#7]
Hi,

In some of the rig setting, the .mat file save `scanbox_version` using the value `sbconfig.firmware`, which is the str(4.0) as you can see in the following screenshot:
![image](https://user-images.githubusercontent.com/65637150/188733543-d4f980e0-3ee6-40c2-8048-6a7a3fa1be1e.png)

Thus, when we do the suite2p registration using .sbx, it raises an error coming from https://github.com/jcouto/sbxreader/blob/main/sbxreader/reader.py#L60

Is it possible to change the if statement a bit? for example:
```
if isinstance(info.scanbox_version, str) or info.scanbox_version < 3:
   um_per_pixel_x = info.calibration[magidx].x
   um_per_pixel_y = info.calibration[magidx].y
```

**Comment by jcouto** [comment]
Hi Yu-Ting,

can you send me a file (the .mat) so I look at what structures that you have? Mail works.

Thanks in advance!

**Comment by ytsimon2004** [comment]
Hi Joao,
here is one of Ben's data recorded from rig2.

best,
Yu-Ting

**Comment by jcouto** [comment]
Got it, had to be cast to float for some reason.
Thanks for the file and help with this! 

I updated master and released version 0.2.1 on pip that fixes this.
